main.go

```diff
@@ -29,63 +22,76 @@ func dbConn() (db *sql.DB) {
 }
 
 func Index(w http.ResponseWriter, r *http.Request) {
+
+	sliceEmployee := []Employee{}
+
 	db := dbConn()
-	selDB, err := db.Query("SELECT *  FROM funcionarios ")
+	result, err := db.Query("SELECT *  FROM employees ")
 	if err != nil {
 		panic(err.Error())
 	}
 
-	n := Names{}
-	res := []Names{}
-
-	for selDB.Next() {
+	for result.Next() {
 		var id int
 		var name, email string
+		var salary float64
 
-		err = selDB.Scan(&id, &name, &email)
+		err = result.Scan(&id, &name, &email, &salary)
 		if err != nil {
 			panic(err.Error())
 		}
 
-		n.Id = id
-		n.Name = name
-		n.Email = email
+		listEmployee := Employee{id, name, email, salary}
```

Comment:
why this variable is called "list" if it holds only one object?

Reply:
I changed it to registeredEmployee